app/helpers/users_helper.rb

```diff
@@ -96,25 +96,32 @@ def build_change_user_status_action(user)
     result
   end
 
+  def change_user_status_icons
+    {
+      "unlock" => "unlocked",
+      "activate" => "unlocked",
+      "lock" => "locked"
+    }
+  end
+
   def change_user_status_buttons(user)
     build_change_user_status_action(user) do |title, name|
-      submit_tag(title, name:, class: "button")
+      button_tag(class: "button", title:) do
+        concat op_icon("icon icon-#{change_user_status_icons[name]}")
```

Comment:
```suggestion
        concat op_icon("button--icon icon-#{change_user_status_icons[name]}")
```

Otherwise the icon will be in the wrong color and the spacing is off